spec/services/payment_providers/adyen_service_spec.rb

```diff
@@ -58,4 +58,87 @@
       end
     end
   end
+
+  describe '#handle_incoming_webhook' do
+    let(:adyen_provider) { create(:adyen_provider, organization:, hmac_key: nil) }
+
+    let(:body) do
+      JSON.parse(event_response_json)['notificationItems'].first&.dig('NotificationRequestItem')
+    end
+
+    let(:event_response_json) do
+      path = Rails.root.join('spec/fixtures/adyen/webhook_authorisation_response.json')
+      File.read(path)
+    end
+
+    before { adyen_provider }
+
+    it 'checks the webhook' do
+      result = adyen_service.handle_incoming_webhook(
+        organization_id: organization.id,
+        body:
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)